ices/86820.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
This doesn't have to be run as an actual crate. All it needs is to place `assert!` on `fn main()`.